docs/sources/administration/planning-iam-strategy/index.md

```diff
@@ -0,0 +1,213 @@
+
+Teams provide more flexibility, as resources can be accessible by multiple teams, and team creation and management are simple.
+
+In contrast, organizations provide more isolation than teams, as resources cannot be shared between organizations.
+They are more difficult to manage than teams, as you must create and update resources for each organization individually.
+Organizations cater to bigger companies or users with intricate access needs, necessitating complete resource segregation.
+
+## Access to external systems
+
+Consider the need for machine-to-machine [M2M](https://en.wikipedia.org/wiki/Machine_to_machine) communications. If a system needs to interact with Grafana, ensure it has proper access.
+
+Consider the following scenarios:
+
+**Schedule reports**: Generate reports periodically from Grafana through the reporting API and have them delivered to different communications channels like email, instant messaging, or keep them in a shared storage.
+
+**Define alerts**: Define alert rules to be triggered when a specific condition is met. Route alert notifications to different teams according to your organization's needs.
+
+**Provisioning file**: Provisioning files can be used to automate the creation of dashboards, data sources, and other resources.
+
+These are just a few examples of how Grafana can be used in M2M scenarios. The platform is highly flexible and can be used in a variety of other M2M applications, making it a powerful tool for organizations looking to gain insights into their systems and devices.
+
+### Service accounts
+
+You can use a service account to run automated workloads in Grafana, such as dashboard provisioning, configuration, or report generation. Create service accounts and service accounts tokens to authenticate applications, such as Terraform, with the Grafana API.
+
+> **Note:** Service accounts will eventually replace [API keys]({{< relref ".#api-keys" >}}) as the primary way to authenticate applications that interact with Grafana.
+
+A common use case for creating a service account is to perform operations on automated or triggered tasks. You can use service accounts to:
+
+- Schedule reports for specific dashboards to be delivered on a daily/weekly/monthly basis
+- Define alerts in your system to be used in Grafana
+- Set up an external SAML authentication provider
+- Interact with Grafana without signing in as a user
+
+In [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise" >}}), you can also use service accounts in combination with [role-based access control]({{< relref "../roles-and-permissions/access-control" >}}) to grant very specific permissions to applications that interact with Grafana.
+
+{{% admonition type="note" %}}
+Service accounts can only act in the organization they are created for. We recommend creating service accounts in each organization if you have the same task needed for multiple organizations.
+{{% /admonition %}}
+
+{{< vimeo 742056367 >}}
```

Comment:
I recommend adding an intro sentence here like "The following video explains blah blah blah."